samples/openvx/openvx_interoperability.cpp

```diff
@@ -0,0 +1,324 @@
+#include <iostream>
+#include <stdexcept>
+
+//OpenVX includes
+#include <VX/vx.h>
+#include <VX/vx_lib_debug.h>
+#include <VX/vx_helper.h>
+#include <VX/vx_lib_extras.h>
```

Comment:
This libraries (`vx_lib_debug`, `vx_helper` and `vx_lib_extras`) are artifacts of sample implementation. They might not be presented in other OpenVX implementations. I would recommend to avoid it.